### PR TITLE
Remove offline resolution CLI variable source

### PR DESCRIPTION
**Pull request by m1kola**
# Description

Spliting #460 into smaller chunks. Related to #437

## Reviewer Checklist

- [ ] API Go Documentation
- [ ] Tests: Unit Tests (and E2E Tests, if appropriate)
- [ ] Comprehensive Commit Messages
- [ ] Links to related GitHub Issue(s)

### CODE REVIEW

File: cmd/resolutioncli/main.go [hunk 1]
```diff
@@ -22,8 +22,7 @@ import (
 	"fmt"
 	"os"
 
-	"github.com/operator-framework/deppy/pkg/deppy/solver"
```
Comment (stevekuznetsov) [inline cmd/resolutioncli/main.go:25]: nit: likely good to use an import sorting linter to remove these diffs in the future

File: cmd/resolutioncli/main.go [hunk 2]
```diff
@@ -71,12 +71,12 @@ func main() {
...
 
 	var packageName string
-	var packageVersion string
+	var packageVersionRange string
 	var packageChannel string
 	var indexRef string
 	var inputDir string
 	flag.StringVar(&packageName, flagNamePackageName, "", "Name of the package to resolve")
-	flag.StringVar(&packageVersion, flagNamePackageVersion, "", "Version of the package")
+	flag.StringVar(&packageVersionRange, flagNamePackageVersionRange, "", "Version of the package")
```
[the remark this answers is not in the document]
Comment (m1kola) [reply cmd/resolutioncli/main.go]: Good catch! Updated.

To be fair: this is not related to removal of varaible soruce. I just noticed that we changed semantics for version field in controller and this was not updated.

It is a tiny change and I didn't bother creating a separate PR. But I can do that if it is too distracting. Let me know.

**Comment by m1kola** [comment]
@stevekuznetsov I see that you enabled auto merge yesterday. Did you also mean to approve it? Or do you have further feedback?

**Comment by stevekuznetsov** [comment]
Sorry, all the repos have different rules for merging and it's too easy to do the wrong dance ...

**Review by stevekuznetsov — APPROVED** [review]
d

**Comment by m1kola** [comment]
> Sorry, all the repos have different rules for merging and it's too easy to do the wrong dance ...

No worries! Thanks for the update!

**Pull request merged**